### DISCUSSION TITLE
Unable to modify exisiting Column

### DISCUSSION THREAD

**Discussion by spacezmonkey** [nocodb/nocodb#556] [Q&A]
Hello all,

i create a column but when i would like to make changes to column type, i receive an warning as shown in my attachment.

May I know if this is normal?  
![22 09 2021_21 24 08_REC](https://user-images.githubusercontent.com/73033222/134368795-655d9f50-ce6f-465c-8604-45bda5af9ccd.png)

**Comment by o1lab** [comment]
NocoDB defaults to sqlite if its not setup for production. And SQLIte columns cant be modified.

Here is how to go about setting up for [production](https://docs.nocodb.com/getting-started/installation#production-setup).

Please check mandatory environment variables required.

Happy to help in our [discord](https://discord.gg/5RgZmkW).
